yarn.lock

```diff
@@ -6984,6 +6991,11 @@ lodash@^4.0.0, lodash@^4.11.1, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.3,
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz#b39ea6229ef607ecd89e2c8df12536891cac9b8d"
   integrity sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==
 
+lodash@^4.17.14:
```

Comment:
Will it make chocapic since we're using `lodash@3.9.3` ?

Reply:
Discusses with @cbourgois and @frenautvh, this is for the module's dep but it won't have impact on current version